src/mixins/chart-element.js

```diff
@@ -92,6 +94,8 @@ export function ChartElement (ParentClass) {
           Highcharts.charts[i].reflow();
         }
       };
+      ChartModuleMore(Highcharts);
+      HCSolidGauge(Highcharts);
```

Comment:
Is everything still working if we move that in the constructor of `src/charts/gv-chart-gauge.js`?
Keeping them here will cause a loading of `highchart-more` + `gauge module` in every app using any Chart component.

Reply:
:+1: